cms/envs/test.py

```diff
@@ -134,6 +134,7 @@
 LMS_BASE = "localhost:8000"
 LMS_ROOT_URL = "http://{}".format(LMS_BASE)
 FEATURES['PREVIEW_LMS_BASE'] = "preview.localhost"
+LOGIN_URL = EDX_ROOT_URL + '/signin'
```

Comment:
I'm guessing some tests rely on this old url? Should we fix the tests somehow to prevent this?

Reply:
Yes, I'm with you on this.  I was planning to do so as a subsequent PR.  Here's the PR I had created yesterday to test exactly this: https://github.com/edx/edx-platform/pull/19465